### PR TITLE
[mds-policy-author] Minor changes to policy to bring it back in line with the spec.

### PR DESCRIPTION
**Pull request by janedotx**
## PR Checklist

 - [ ] simple searchable title - `[mds-db] Add PG env var`, `[config] Fix eslint config`
 - [ ] briefly describe the changes in this PR
 - [ ] mark as draft if should not be merged
 - [ ] write tests for all new functionality

## Impacts
- [ ] Provider
- [ ] Agency
- [ ] Audit
- [ ] Policy
- [ ] Compliance
- [ ] Daily
- [ ] Native
- [ ] Policy Author

### CODE REVIEW

File: packages/mds-geography/package.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "@mds-core/mds-utils": "0.1.26",
     "express": "4.17.1",
     "sinon": "7.5.0",
-    "uuid": "7.0.1"
+    "uuid": "7.0.2"
```
Comment (avatarneil) [inline packages/mds-geography/package.json]: Surprised this didn't update the `yarn.lock` file

File: packages/mds-types/index.ts [hunk 2]
```diff
@@ -453,6 +453,10 @@ export interface AuditDetails extends Audit {
   }
 }
 
+export interface PolicyMessage {
```
Comment (avatarneil) [inline packages/mds-types/index.ts]: Seems like a kinda weird type to define, but even if we want to, it doesn't need to be an exported type for sure

File: packages/mds-geography/package.json (hunk 1 above)
Comment (janedotx) [reply packages/mds-geography/package.json]: All the other packages are on 7.0.2.

File: packages/mds-types/index.ts (hunk 2 above)
Comment (janedotx) [reply packages/mds-types/index.ts]: Well the spec does say it has to be of the form `{ 'language_code': 'message' }`, so might as well, but I can def remove the `export`

**Pull request merged**